components/Graph/Graph.tsx

```diff
@@ -13,10 +13,10 @@ import { Contact } from "../../types/Contact"
  */
 interface Node {
   id: string
-  x: number
-  y: number
-  dx: number
-  dy: number
+  x?: number
+  y?: number
+  fx?: number | null
+  fy?: number | null
```

Comment:
not sure you need the | null with the syntax

Reply:
because it is the way to interact with d3.js

Reply:
yes but i think the ? in the name of the parameter already means that the argument can be null

Reply:
no, they mean undefined